docs/api/README.md

```diff
@@ -78,31 +78,16 @@
 Creates a Redux middleware and connects the Sagas to the Redux Store
 
 - `options: Object` - A list of options to pass to the middleware. Currently supported options are:
+  - `context: Object` - initial value of the saga's context.
 
   - `sagaMonitor` : [SagaMonitor](#sagamonitor) - If a Saga Monitor is provided, the middleware will deliver monitoring events to the monitor.
 
- - `emitter` : Used to feed actions from redux to redux-saga (through redux middleware). Emitter is a higher order function, which takes a builtin emitter and returns another emitter.
-
-      **Example**
-
-      In the following example we create an emitter which "unpacks" array of actions and emits individual actions extracted from the array.
-
-     ```javascript
-     createSagaMiddleware({
-       emitter: emit => action => {
-        if (Array.isArray(action)) {
-          action.forEach(emit);
-          return
-        }
-        emit(action);
-       }
-     });
-     ```
-
   - `logger` : Function -  defines a custom logger for the middleware. By default, the middleware logs all errors and
 warnings to the console. This option tells the middleware to send errors/warnings to the provided logger instead. The logger is called with the params `(level, ...args)`. The 1st indicates the level of the log ('info', 'warning' or 'error'). The rest corresponds to the following arguments (You can use `args.join(' ')` to concatenate all args into a single String).
 
   - `onError` : Function - if provided, the middleware will call it with uncaught errors from Sagas. useful for sending uncaught exceptions to error tracking services.
+  - `effectMiddlewares` : Function [] - allows you to intercept any effect, resolve it on your own and pass to the next middleware.
```

Comment:
Here we could add a link to the testing docs which contains a detailed example of `effectMiddlewares`.

We could add this line: `See [this section](/docs/advanced/Testing.md#effectmiddlwares) for a detailed example.`